pdg/src/info.rs

```diff
@@ -114,25 +115,26 @@ fn check_children_conflict(
     children: &HashMap<NodeId, Vec<NodeId>>,
     descs: &HashMap<NodeId, NodeId>,
 ) -> bool {
-    let mut max_descs: HashMap<Option<Field>, NodeId> = HashMap::new();
-    for id in children.get(n_id).unwrap() {
-        let sib_node: &Node = g.nodes.get(*id).unwrap();
-        let my_last_desc = *descs.get(id).unwrap();
-        // if the first below matches, then two siblings, neither a field, conflict
-        // if the second matches, then two siblings of the same field conflict
-        if matches!(max_descs.get(&None), Some(max_desc) if max_desc > id)
-            || matches!(sib_node.kind, NodeKind::Field(f) if matches!(max_descs.get(&Some(f)), Some(max_desc_field) if max_desc_field > id))
-        {
+    let mut max_descs = HashMap::new();
+    for id in &children[n_id] {
```

Comment:
in another PR you made `id` clearer by renaming to `child_id`. I think similar changes should happen here

Reply:
Okay, I'll try to make them all consistent.  I've been using `child` and `parent` for the `NodeId`s, and `child_node` for the `Node` (though there are a couple of exceptions, which I'll fix).  Using the `NodeId` is much more common so I'm using the shorter name for them.

Reply:
Made consistent in:
* https://github.com/immunant/c2rust/pull/669/commits/a1dfd02f40a1c501a9342460914aa96cf5c7d728
* https://github.com/immunant/c2rust/pull/671/commits/ec0e70a9285f6c6d452e03b18bbc2a4e0dc27d2f